lib/middleware/service.js

```diff
@@ -29,6 +29,36 @@ var logger = require('logops'),
     paths = require('../../lib/models/paths'),
```

Comment:
CNR entry missing. Maybe we could even include a mention to recently closed #128. To be discussed by skype.